AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -7,22 +7,14 @@ import Foundation
 import AzureCommunicationCalling
 import AVFoundation
 
-enum CallingInterfaceState {
-    case connecting
-    case waitingAdmission
-    case admissionDenied
-    case connected
-    case removed
-}
-
 public typealias TokenFetcher = (@escaping (String?, Error?) -> Void) -> Void
 
 class CallingContext: NSObject {
     // MARK: Constants
     private static let remoteParticipantsDisplayed: Int = 3
 
     // MARK: Properties
-    private (set) var joinId: String!
+    private (set) var groupId: String!
```

Comment:
We can continue to use `joinId`. There will be less conflicts when we merge later on.